python/samples/agentchat_graphrag/app.py

```diff
@@ -0,0 +1,67 @@
+import asyncio
+from autogen_agentchat.messages import TextMessage
+from autogen_ext.models.openai import AzureOpenAIChatCompletionClient
+from autogen_ext.tools.graphrag import (
+    GlobalSearchTool,
+    LocalSearchTool,
+)
+from azure.identity import DefaultAzureCredential, get_bearer_token_provider
+from autogen_agentchat.agents import AssistantAgent
+from autogen_agentchat.conditions import TextMentionTermination, MaxMessageTermination
+from autogen_agentchat.teams import RoundRobinGroupChat
+
+
+async def main():
+    # Initialize the OpenAI client
+    openai_client = AzureOpenAIChatCompletionClient(
```

Comment:
With the recent addition of component configuration now we can use a config file to specify the endpoints. 

See another sample: https://github.com/microsoft/autogen/tree/main/python/samples/core_chess_game